index.js

```diff
@@ -0,0 +1,104 @@
+const playerContainer = document.querySelector(".player-container");
+const playerForm = document.getElementById("new-player-form");
+const url = "http://localhost:3000/players";
+
+function init() {
+  fetch(url)
+    .then((response) => response.json())
+    .then((data) => {
+      data.forEach((player) => {
+        renderPlayer(player);
+      });
+    });
+}
+function renderPlayer(playerObj) {
+  const player = createNode(
+    "h3",
+    `${playerObj.name} (<em>${playerObj.nickname} 
+        - Number: ${playerObj.number}</em>) 
+        Likes: ${playerObj.likes} `
+  );
+  player.className = "playerObj";
+  player.style.listStyle = "none";
+  const likeBtn = createNode("button", "❤️");
+  player.append(addLikeEvent(likeBtn, playerObj));
```

Comment:
This line is a bit hard to follow, I might suggest just using your `addLikeEvent` on its own line for clarity:

```js
  const likeBtn = createNode("button", "❤️");
  addLikeEvent(likeBtn, playerObj);
  player.append(likeBtn);
```